lib/http-conditional.php

```diff
@@ -142,13 +142,13 @@ function httpConditional(int $UnixTimeStamp, int $cacheSeconds = 0, int $cachePr
 	}
 
 	if ($is412) { //rfc2616-sec10.html#sec10.4.13
-		header('HTTP/1.1 412 Precondition Failed');
+		header(FreshRSS_HttpResponse::description(FreshRSS_HttpResponse::HTTP_412_PRECONDITION_FAILED));
```

Comment:
This file is a standalone library, which does not know about FreshRSS_HttpResponse...